actions/templates_test.go

```diff
@@ -0,0 +1,29 @@
+package actions
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetTemplates(t *testing.T) {
+	tests := map[string]struct {
+		wantedType   []Template
+		wantedLength int
+		wantedErr    error
+	}{
+		"success case: no input": {
+			wantedType:   []Template{},
+			wantedLength: 8,
```

Comment:
May become an issue when templates are added? Could we mock this list when we get time, which will be useful for all future template commands too?